src/mqt/bench/cli.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+import argparse
+
+from pytket import Circuit
+from pytket.qasm import circuit_to_qasm_str
+from qiskit import QuantumCircuit
+from qiskit.qasm2 import dumps as qiskit_circuit_to_str
+
+from mqt.bench import CompilerSettings, QiskitSettings, TKETSettings, get_benchmark
+
+
+def main() -> None:
+    """
+    Invoke :func:`get_benchmark` exactly once with the specified arguments.
+
+    The resulting QASM string is printed to the standard output stream;
+    no other output is generated.
+    """
+
+    parser = argparse.ArgumentParser(description="Generate a single benchmark")
+    parser.add_argument(
+        "--level",
+        type=str,
+        help='Level to generate benchmarks for ("alg", "indep", "nativegates" or "mapped")',
+        required=True,
+    )
+    parser.add_argument("--algorithm", type=str, help="Name of the benchmark", required=True)
+    parser.add_argument("--num-qubits", type=int, help="Number of Qubits")
+    parser.add_argument("--compiler", type=str, help="Name of the compiler")
+    parser.add_argument("--qiskit-optimization-level", type=int, help="Qiskit compiler optimization level")
+    parser.add_argument("--tket-placement", type=str, help="TKET placement")
+    parser.add_argument("--native-gate-set", type=str, help="Name of the provider")
+    parser.add_argument("--device", type=str, help="Name of the device")
+    args = parser.parse_args()
+
+    qiskit_settings: QiskitSettings | None = None
```

Comment:
```suggestion
    qiskit_settings = QiskitSettings()
```
This sets the default values properly.